dev_tools/scripts/update_ginkgo_header.sh

```diff
@@ -1,7 +1,17 @@
 #!/usr/bin/env bash
 
+# Note: This script is supposed to support the developer and not to hinder
+#       the development process by setting more restrictions.
+#       This is the reason why every exit code is 0, otherwise, the whole
+#       `make` procedure would fail.
+
 PLACE_HOLDER="#PUBLIC_HEADER_PLACE_HOLDER"
 
+WARNING_PREFIX="[WARNING] ginkgo.hpp update script failed because:"
+
+RM_PARAMETER="--interactive=never"
```

Comment:
Weird, my comment was lost.

I fear this is a `GNU rm` only parameter and therefore not available on MacOS. I'm not sure about Windows. You can use `-f` here since this should be available everywhere and no `-r` parameter is passed therefore even if the content of the variable has the wrong file, it should not have horrible results for the user.  

See:
https://stackoverflow.com/questions/36551173/how-to-auto-cancel-deleting-while-rm-asking-for-confirm/36551587#36551587
https://ss64.com/osx/rm.html

Reply:
Thanks, I did not know that this parameter was GNU only.